com.unity.render-pipelines.universal/Runtime/ForwardRendererData.cs

```diff
@@ -129,7 +141,46 @@ public bool shadowTransparentReceive
             }
         }
 
-        protected override void OnEnable()
+        /// <summary>
+        /// Use deferred shading path for compatible lights.
+        /// </summary>
+        public bool deferredShading
+        {
+            get => m_DeferredShading;
+            set
+            {
+                SetDirty();
+                m_DeferredShading = value;
+            }
+        }
+
+        /// <summary>
+        /// Use Octaedron Octahedron normal vector encoding for gbuffer normals.
+        /// The overhead is negligible from desktop GPUs, while it should be avoided for mobile GPUs.
```

Comment:
is it something that we should expose to users if it should be avoided in some platforms? If we expose something like this they will expect to work in all platforms that support deferred.

Reply:
@phi-lira  At the moment, not using octahedron encoding will generate banding artefact on smooth surfaces like a sphere. However, it is much cheaper to not use octahedron encoding on mobile GPUs. I think there should be an "auto" setting that disable it by default on mobile GPUs.

Should we replace by an enum with { disable, enable, auto } values instead?

Reply:
There was a recent discussion about where to store settings that are related to quality. If stored in the pipeline asset then use can configure one asset per quality settings and we can also provide good defaults with the template project for what it means for each platforms. Maybe this can be asked in strike-team channel.

Reply:
I am leaving things as is for now and revisit after this PR merges (UI for deferred not enabled yet). Does URP has quality settings exposed now?